package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@qonto/eslint-config-typescript",
+  "version": "0.0.1",
+  "description": "ESLint TypeScript configuration.",
+  "scripts": {
+    "test": "NODE_OPTIONS=--experimental-vm-modules pnpm exec jest"
```

Comment:
The `--experimental-vm-modules` is needed [because of this](https://stackoverflow.com/questions/74069138/node-js-experimental-vm-modules-command-line-option-vs-type-module-in-pac), right?

Reply:
Exactly. And I had a lot of issues setting it up. I followed the [documentation from Jest](https://jestjs.io/docs/ecmascript-modules). This flag is recommended

Reply:
Clear, thanks! 🙏